integration/mimir/testdata/test-rules-1.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: grizzly.grafana.com/v1alpha1
+kind: RuleGroup1
```

Comment:
doesn't `RuleGroup1` confuse Grizzly? Shouldn't it be `PrometheusRuleGroup`? Same for `RuleGroup2` and `RuleGroup3` below.